core/src/scss/utilities/mixins/link/_external-link.scss

```diff
@@ -13,17 +13,22 @@
 @mixin external-link($external-link: external-link, $external-link-hover: external-link-hover) {
   &::after {
     @include margin(null null -1px 4px);
-    background: url("#{$image-path}/#{$external-link}.png") no-repeat 0 0;
-    background: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
-    background-size: 100%;
-    content: '';
+    @include size(0.65em);
     display: inline-block;
-    height: 0.65em;
-    width: 0.65em;
+    content: '';
+    mask: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
```

Comment:
Browser support for this looks limited: https://caniuse.com/#feat=css-masks
Did you check that this spec meets our compatibility requirements? https://github.com/SU-SWS/decanter/wiki/Compatibility